tutorials/focus-mode/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "manifest_version": 3,
+  "name": "Focus Mode",
+  "description": "Enable reading mode on extension and web store documentation pages.",
+  "version": "1.0",
+  "icons": {
+    "16": "images/icon-16.png",
+    "32": "images/icon-32.png",
+    "48": "images/icon-48.png",
+    "128": "images/icon-128.png"
+  },
+  "background": {
+    "service_worker": "background.js"
+  },
+  "action": {
+    "default_icon": {
+      "16": "images/icon-16.png",
+      "32": "images/icon-32.png",
+      "48": "images/icon-48.png",
+      "128": "images/icon-128.png"
+    }
+  },
+  "permissions": ["scripting", "activeTab", "commands"],
```

Comment:
According to the [Declare permissions] page, there is no `commands` permission.

[Declare permissions]: https://developer.chrome.com/docs/extensions/mv3/declare_permissions/

Surprisingly, this key does not trigger an error, but if you replace `commands` with `foo`, it gives:

```
Permission 'foo' is unknown or URL pattern is malformed.
```

Reply:
@alexherbo2 Thanks for pointing this out. I just tested it, and it works without including the commands permission 🤔